content/shmem_malloc_hints.tex

```diff
@@ -44,6 +44,17 @@
     The user is also responsible for calling these routines with identical argument(s) on all
     \acp{PE}; if differing \VAR{size}, or \VAR{hints} arguments are used, the behavior of the call
     and any subsequent \openshmem calls is undefined.
+
+	Calling the routine \FUNC{shmem\_realloc(void *ptr, size\_t size)} on the
+	ptr allocated by \FUNC{shmem\_malloc\_with\_hints} has a behavior similar to
+	calling \FUNC{shmem\_realloc} on the ptr allocated by \FUNC{shmem\_malloc}.
+	If the new size is larger, the memory is allocated with the same hints that
+	were passed when calling \FUNC{shmem\_malloc\_with\_hints}, and the newly
+	allocated portion of the block is uninitialized. If ptr is a null pointer,
```

Comment:
Why not just make this wording consistent with C realloc and say "the contents of the new part of the array are undefined"?